README.md

```diff
@@ -14,7 +15,43 @@ CodeFlare Stack Compatibility Matrix
 | KubeRay                      | v0.5.0  |
 <!-- Compatibility Matrix end -->
 
-## Release process
+## Development
+
+### Testing
+
+The e2e tests can be executed locally by running the following commands:
+
+1. Use an existing cluster, or set up a test cluster, e.g.:
+
+    ```bash
+    # Create a KinD cluster
+    $ kind create cluster --image kindest/node:v1.25.8
+    # Install the CRDs
+    $ make install
+    ```
+
+2. Set up the CodeFlare stack:
+   ```bash
+   $ make setup-e2e
+   ```
+
+3. Start the operator locally:
+
+    ```bash
+    $ make run
+    ```
```

Comment:
I think these should be swapped. Make setup-e2e assumes that the operator is running so that it can create an MCAD deployment

Reply:
Ah right, or the `kubectl wait` command that waits for the MCAD deployment to become ready can be removed. It's not strictly necessary.

Reply:
Done with #163.

Reply:
Thanks!